k8/base/cronjob.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: hello
+spec:
+  schedule: "0 12 1 * *"
```

Comment:
This should be something like 8.30am rather than midnight to give time to the other invoicing scripts to finish their job and place everything in S3.

Reply:
Sorry, I made a typo, forgetting that "12AM meant midnight. " `0 12 1 * *` would mean running at Noon, `0 0 1 * *` would have been midnight

Reply:
I run my cronbjo on the nerc cluster and those are all doing things in UTC. So, this would be 12noon at UTC or 8am boston time if run on a nerc openshift cluster.

FWIW, my cronjob produces the openshift report at 9 am Boston time. I am happy to change that if you want, but this should run after the openshift, openstack and coldfront reports are uploaded. 

I'd vote to change it to produce the final report at 10am Boston time or 2pm UTC, so `0 14 1 * *`.

Reply:
Should I specify a Boston timezone in the cronjob manifest?